components/date-picker/locale/nb_NO.tsx

```diff
@@ -1,17 +1,53 @@
-import CalendarLocale from 'rc-picker/lib/locale/nb_NO';
 import TimePickerLocale from '../../time-picker/locale/nb_NO';
 import { PickerLocale } from '../generatePicker';
 
 // Merge into a locale object
 const locale: PickerLocale = {
   lang: {
+    locale: 'nb_NO',
     placeholder: 'Velg dato',
+    yearPlaceholder: 'Velg år',
+    quarterPlaceholder: 'Velg kvartal',
+    monthPlaceholder: 'Velg måned',
+    weekPlaceholder: 'Velg uke',
     rangePlaceholder: ['Startdato', 'Sluttdato'],
-    ...CalendarLocale,
```

Comment:
revert this

Reply:
Updated. All the placeholders are not in rc-picker, but is in the other translations. So keeping them unless you think they also should go